st2common/st2common/config.py

```diff
@@ -210,7 +210,8 @@ def register_opts(ignore_errors=False):
         cfg.StrOpt('api_url', default=None,
                    help='Base URL to the API endpoint excluding the version'),
         cfg.BoolOpt('enable', default=True, help='Enable authentication middleware.'),
-        cfg.IntOpt('token_ttl', default=86400, help='Access token ttl in seconds.')
+        cfg.IntOpt('token_ttl', default=86400, help='Access token ttl in seconds.'),
+        cfg.IntOpt('service_token_ttl', default=86400, help='Service token ttl in seconds.')
```

Comment:
Would be good to add a comment which clarifies a bit what service token is and where it's used for (basically what we debated in this PR :D)